### DISCUSSION TITLE
"Detection of the cosmological time dilation of high-redshift quasars"

### DISCUSSION THREAD

**Discussion by HanDeBruijn** [a-cosmology-group/acg#96] [Astrophysics and Physics]
[Time appeared to move 5 times more slowly in 1st billion years after Big Bang, quasar 'clocks' reveal](https://www.space.com/quasar-clocks-universe-time-dilation) :
_Time dilation, brought about by the relativistic expansion of space, has resulted in the observed slowing of 'clocks' in the early universe._

With the [UAC](https://github.com/orgs/a-cosmology-group/discussions/63#discussioncomment-6255770) version of the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs) we derive the following. **Without** any _relativistic expansion of space_.

$$
\frac{\Delta t}{\Delta t_0} = \frac{m_0}{m} = 1+z
$$

where $\Delta t=$ size of atomic clock tick on quasar, $\Delta t_0=$ size of atomic clock tick on earth, $m=$ electron mass on quasar, $m_0=$ electron mass on earth, $z=$ cosmic redshift of quasar. Now let the atomic clocks at the quasar indeed _move 5 times more slowly_ than the atomic clocks on earth, then we have

$$
\frac{\Delta t}{\Delta t_0} = 5 \quad ; \quad z = 4 \quad ; \quad \frac{m}{m_0} = \frac{1}{5}
$$

In the [UAC](https://github.com/orgs/a-cosmology-group/discussions/63#discussioncomment-6255770) reference, we find a formula called Narlikar's law:

$$
\frac{m}{m_0} = \left[1+\frac{H}{2}(T-T_0)\right]^2 \quad \Longrightarrow \quad \frac{1}{\sqrt{5}} = 1+\frac{H}{2}(T-T_0)
$$

where $T=$ orbital time and $H=$
1. 'Hubble constant -- at 67.4 (+/- 0.5) km/sec/Mpc', NASA Science, July 22, 2018
2. 'Hubble constant -- at 73.2 (+/- 1.3) km/sec/Mpc', The Astrophysical Journal Letters, February 2021

Remember that there are no Bangs in Atomic time frames $t$ , so if there __is__ a (Big) Bang, then it can only happen in an Orbital time frame $T$. Anyway, the (orbital) back in time (age) of our quasar can be calculated:

$$
T_0-T = 2\left(1-\frac{1}{\sqrt{5}}\right)\times\frac{1}{H}
$$

```
year := 31556926;
Mpc := 3.08567758*10^22;
H := 73.2*1000/Mpc;
ago := evalf(2*(1-1/sqrt(5))/H/(year*10^9));

                          ago := 14.7(6835494)
```
If the _Big Bang has never happened_ 13.7 billion years ago, then **our quasar turns out to be older than the age of the universe**. One billion years indeed, but the other way around !!

_"At its heart, this is another 'Einstein is right again' story," Geraint Lewis, a cosmologist at the University of Sydney, told Space.com._ **Really?**
It is demonstrated here that **[General Relativity](https://github.com/orgs/a-cosmology-group/discussions/78) is not needed at all** to explain _the cosmological time dilation of high-redshift quasars_.

**Comment by MJoseph315** [comment]
Interesting comments, Han. Is there any other way to explain this supposed evidence of quasar time dilation without relying on a variable mass theory? I am asking the whole group here (Sahil, Louis, etc.)

**Reply by RedshiftDrift** [reply]
There are many other ways to explain apparent "time dilation", many of [these models](https://cosmology.info/models/) account for that.
These explanations also account for supernovae light-curve dilation.

However, the explanation should also account for [***no apparent time dilation*** of gamma ray bursts](http://arxiv.org/abs/2212.08096) and [***no apparent time dilation of Fast Radio Bursts*** arXiv:2101.07998](https://arxiv.org/abs/2101.07998) (@HanDeBruijn ?)

I don't know of any published work that explains all these observations.

**Reply by HanDeBruijn** [reply]
@MJoseph315 
Somewhat to my surprise, the original paper in _Nature Astronomy_ has become available now. Allow me some nitpicking.
Their research _comprises 190 quasars, covering the redshift range_ $z \approx 0.2-4.0$ .
_In the following analysis, we consider the redshift dependence of time dilation to be of the form_ $(1+z)^n$ _, where_ $z$ _is the redshift of the source._
_One immediate conclusion is that the favoured hypothesis is_ [ .. ] _the case where_ $n=1$ [ .. ]

These statements are in concordance with theory as presented above: $z=4.0$ and time dilation is proportional to $(1+z)^1$ .
The authors however are experimentalists and do not present any cosmological theory. They simply accept the standard explanation, saying that their results are _again consistent with the expected cosmological expansion of space_. So if you want an explanation _without relying on a variable mass theory_, you __could__ simply adhere to standard cosmology, as they do.

**Reply by HanDeBruijn** [reply]
@marmetl  . With the Variable Mass Theory, _the Anti-Correlation between Duration and Redshift in **Gamma-ray Bursts**_ has an extremely simple explanation. Take a look at the picture below.
![klokHdB](https://github.com/a-cosmology-group/acg/assets/135433222/683c6c09-fb8f-4670-9454-f6428679d436)
The secret is in the atomic clock ticks, which are different on Earth and where the GRB originates. We repeat:

$$
\large \frac{\Delta t}{\Delta t_0} = \frac{m_0}{m} = 1+z
$$

But we are measuring the burst in Earthly seconds, not in GRB seconds "as it should". With the latter seconds, the GRBs are "normal". They are shortened by the fact that our atomic clocks simply run "too fast". So indeed, as established empirically in the paper:

$$
\large \frac{GRB_{obs}}{GRB_{true}} = (1+z)^{-1}
$$

**Reply by RedshiftDrift** [reply]
> They are shortened by the fact that our atomic clocks simply run "too fast".

Could you have this backwards @HanDeBruijn ?  If our "atomic clock runs fast", a **larger number** is indicated on the clock.  Therefore we say "this took **more** time".  I believe you wanted to write

$$
\frac{GRB_{obs}}{GRB_{int}} = 1+z {\mbox{\hspace{30pt}???}}
$$

with the subscript $int$ for *intrinsic* (the word *true* is not allowed in physics).

---

Reading the [paper](http://arxiv.org/abs/2212.08096) carefully... "Figure 1 *Intrinsic gamma-ray duration* ***(corrected for cosmological time dilation)***" shows

$$
GRB_{int} = constant \times (1+z)^{-1}.
$$

Without the correction for ΛCDM cosmology, this means

$$
GRB_{obs} = constant.
$$

All selectd GRBs are assumed to be similar clocks so that $GRB_{int} = constant$ and

$$
\frac{GRB_{obs}}{GRB_{int}} = 1 \ne (1+z)^{-1} \ne (1+z)
$$

which doesn't agree with your equation (nor with the corrected version of it).

---

The challenge is to explain why some "clocks" (supernova (SN) and quasar (QSO)) show time dilation

$$
\large \frac{SN_{obs}}{SN_{int}} = \frac{QSO_{obs}}{QSO_{int}} = 1+z,
$$

but other "clocks" (Gamma Ray Burst (GRB) and (possibly) Fast Radio Burst (FRB)) don't show time dilation

$$
\large \frac{GRB_{obs}}{GRB_{int}} = \frac{FRB_{obs}}{FRB_{int}} = 1.
$$

**Reply by HanDeBruijn** [reply]
@marmetl .
Sabine Hossenfelder can explain it more clearly than I ever could: [Time Ran Slower in Early Universe, New Study Finds](https://www.youtube.com/watch?v=66V4RSmDqYM) , first three minutes only.

Concerning the Gamma-Ray Bursts. It's basically the same issue as with [Length Contraction](http://www.alternatievewiskunde.nl/cosmology/lengte.htm) and [Time Dilation](http://www.alternatievewiskunde.nl/cosmology/klokken.htm) in **Special** Relativity. Why is the first one a multiplication ($L=$ length) and the second one a division ($t=$ time) by the same factor?

$$
\Delta L = \Delta L_0 \times \sqrt{1-(v/c)^2} \quad ; \quad \Delta t = \Delta t_0 / \sqrt{1-(v/c)^2}
$$

_Without the correction_, you say? Standard cosmology with expanding space is **equivalent** with the variable mass theory, by means of a transformation in General Relativity invented by Jayant Narlikar. So the same "correction" applies, I think. Apart from this, why don't they publish their empirical results without the pollution by theoretical constructs? That would save us from quite some misunderstanding.

_All selected GRBs are assumed to be similar clocks_. Yes, but what is "similar". And your subscript "int" might have another meaning in the variable mass theory. It rather should be something like "in proper place and in proper time" at the Gamma Burst itself if you want to avoid the (what's in a) name "true".

**Reply by HanDeBruijn** [reply]
@marmetl . Conclusion:
> (Gamma Ray Burst (GRB) and (possibly) Fast Radio Burst (FRB)) don't show time dilation

$$
\large \frac{GRB_{obs}}{GRB_{int}} = \frac{FRB_{obs}}{FRB_{int}} = 1.
$$

This means that our picture should be adapted as follows. And the Time Dilation correction shouldn't be applied.
![klokLM](https://github.com/a-cosmology-group/acg/assets/135433222/6dfb1d4e-a334-4095-89d8-4b2ed3c8ab79)
The picture on the left (mine) would involve sort of cosmic fine-tuning if the gamma bursts really belong to different galaxies. The picture on the right (Louis?) doesn't need that. Furthermore, the redshift mechanism on the right cannot be Variable Mass, so it must be something else (such as Tired Light). Question is: **do the Gamma Rays themselves show Cosmic Redshift?**

**Reply by RedshiftDrift** [reply]
**Observed after correction** doesn't make sense.  One observes ***first***, then one makes corrections...

> ... do the Gamma Rays themselves show Cosmic Redshift?

It's not clear, gamma rays have no reference spectral lines and they don't seem to have a frequency that correlates with properties of the galaxies from which they originate [https://iopscience.iop.org/article/10.3847/1538-4357/aa5fa7](https://iopscience.iop.org/article/10.3847/1538-4357/aa5fa7)

**Comment by MJoseph315** [comment]
@marmetl
Which of these models specifically, can explain the effect? And can those models explain these new results? I have heard that some evidence implied that quasars did not show the time dilation expected in the expanding universe model. Are there any potential flaws or arbitrary assumptions in this study that may have biased the results?

**Reply by RedshiftDrift** [reply]
Every paper based on the expanding space model is flawed because the interpretation of redshift is limited to
- space expansion,
- gravitational time dilation, and
- a Doppler effect.

An additional redshift mechanism is at play, Edwin Hubble called it "a new principle of nature" in 1937.
